tools/get_quad_ids.py

```diff
@@ -0,0 +1,52 @@
+from penquins import Kowalski
+import json
+
+
+def get_field_ids(catalog, field=301, ccd=2, quad=3, minobs=5, n=10):
```

Comment:
We also need some kind of skiplimit argument so we can cycle through the field to grab all IDs?